services/frontend/src/components/qualifiedPools/QualifiedPoolsView.scss

```
@@ -0,0 +1,13 @@
+.qualificationPools-Link {
+  padding-left: 0px !important;
+}
+
+.qualificationPools-Link svg {
+  margin-right: 5px !important;
+}
+
+@media (min-width: 780px) {
+  .qualificationPools-Link {
+    padding-left: 20px !important;
+  }
+}
```

This can be simplified with scss

```suggestion
.qualificationPools-Link {
  padding-left: 0px !important;
  
  svg {
    margin-right: 5px !important;
  }
  
  @media (min-width: 780px) {
    padding-left: 20px !important;
  }
}
```

---

Will do!